fetch-schema.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+require('dotenv').config({ path: '.env.local' })
+const exec = require('child_process').exec
```

Comment:
turbo nit: ` const { exec } = require('child_process')`

Reply:
I'll change it in prei18n too then